src/widgets/MenuItem.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+import { MaterialCommunityIcons } from "@expo/vector-icons";
+import { Menu } from "react-native-paper";
+import { useTheme } from "../theme";
+
+type PropsType = Omit<React.ComponentProps<typeof Menu.Item>, "icon"> & {
+  icon?: string;
+  active?: boolean;
+};
+
+export default function MenuItem(props: PropsType) {
+  const { active, disabled, icon, ...rest } = props;
+  const { colors } = useTheme();
+  
+  return (
+    <Menu.Item
+      disabled={disabled}
+      icon={(icon) ? ({ color, size }) => <MaterialCommunityIcons name={icon} size={size} color={(active && !disabled) ? colors.activeIcon : color} /> : undefined}
+      theme={(active) ? { colors: { text: colors.active } } : undefined}
+      style={(active) ? { backgroundColor: colors.activeBackground } : undefined}
```

Comment:
You should be merging your style with the style passed in props. Same I guess for the theme (though not as important). At the moment this widget fails where there's a style passed to it (the style passed will just override yours completely).

Reply:
The point was to use this style and theme, because if we want to have a consistent app we should style a component for every use case we have and reuse it as-is.

Maybe I should remove the theme and style props to make that clear.

Reply:
I understand this is the point, but it's currently in the type so people expect it to work.
Also, some styles may be useful still to set, so maybe just merge the styles. It's very easy to do, as the `style` prop accepts an array of styles that override each other...